Apps/WebClient/src/NewClientApp/src/views/HomeView.vue

```diff
@@ -1,7 +1,541 @@
+            });
+            res.blob().then((blob) => saveAs(blob, "VaccineProof.pdf"));
+        });
+        stopAuthenticatedVaccineRecordDownload(user.value.hdid);
+    }
+});
+</script>
 
 <template>
-    <h1>home</h1>
+    <v-container>
+        <LoadingComponent
+            :is-loading="isVaccineRecordDownloading"
+            :text="vaccineRecordStatusMessage"
+        />
+        <v-alert
+            v-if="unverifiedEmail || unverifiedSMS"
+            data-testid="incomplete-profile-banner"
+            show
+            dismissible
+            type="info"
+            class="no-print my-3"
+            title="Verify Contact Information"
+        >
+            <span class="text-body-1">
+                Your email or cell phone number has not been verified. You can
+                use the Health Gateway without verified contact information,
+                however, you will not receive notifications. Visit the
+                <router-link
+                    id="profilePageLink"
+                    data-testid="profile-page-link"
+                    variant="primary"
+                    to="/profile"
+                    >Profile Page</router-link
+                >
+                to complete your verification.
+            </span>
+        </v-alert>
+        <v-alert
+            v-if="!isPacificTime"
+            show
+            closable
+            type="info"
+            title="Looks like you're in a different timezone."
+            text="Heads up: your health records are recorded and displayed in
+                Pacific Time."
+            class="no-print my-3"
+        />
+        <PageTitleComponent title="Home">
+            <template #append>
+                <AddQuickLinkComponent
+                    :disabled="isAddQuickLinkButtonDisabled"
+                />
+            </template>
+        </PageTitleComponent>
+        <v-row>
+            <v-col :cols="quickLinkCols">
+                <HgCardComponent
+                    title="Health Records"
+                    data-testid="health-records-card"
+                    @click="handleClickHealthRecords()"
+                    class="h-100"
+                    elevation="6"
+                >
+                    <template #icon>
+                        <img
+                            class="quick-link-icon align-self-center"
+                            src="@/assets/images/gov/health-gateway-logo.svg"
+                            alt="Health Gateway Logo"
+                        />
+                    </template>
+                    <p class="text-body-1">
+                        View and manage all your available health records,
+                        including dispensed medications, health visits, COVID‑19
+                        test results, immunizations and more.
+                    </p>
+                </HgCardComponent>
+            </v-col>
+            <v-col :cols="quickLinkCols" v-if="showFederalCardButton">
```

Comment:
Nit: as a style preference, I prefer `v-if` directives (and `v-else`, `v-else-if`, `v-for`) to be listed before other props and directives, to more easily track control flow in the template.